Sources/TripKit/model/TKBookingTypes.swift

```diff
@@ -65,8 +66,29 @@ public enum TKBooking {
       case internalURL
       case externalAction = "externalURL"
       case type
-      case input
       case confirmationMessage
```

Comment:
Once the new `confirmation` field is live in production, we can also get rid off `confirmationMessage` or mark it as deprecated in the SDK.

Reply:
added deprecation notice